package.json

```
@@ -72,10 +72,10 @@
     "@babel/preset-env": "^7.3.4",
     "@babel/preset-react": "^7.0.0",
     "@emotion/core": "^10.0.7",
-    "@storybook/addon-options": "~5.1.11",
-    "@storybook/addon-storysource": "~5.1.11",
-    "@storybook/addons": "~5.1.11",
-    "@storybook/react": "~5.1.11",
+    "@storybook/addon-options": "~5.3.18",
```

would it work if we'll use the caret?
```suggestion
    "@storybook/addon-options": "^5.3.18",
```

---

let me check

---

I think theres no problem using `^`

---

Unless storybook releases a breaking change out of the blue (which is very improbable), I see no issue, im updating this rn